### PR TITLE
README.md improvements

### PR DESCRIPTION
**Pull request by techtonik**

### CODE REVIEW

**Comment by carlkl** [comment]
sounds good to me.

**Comment by techtonik** [comment]
That was quick. =) I actually started this to merge both readme files together, which is now at  #16

File: README.md [hunk 1]
```diff
@@ -1,8 +1,7 @@
-*mingwpy* is a customized version of the [GNU toolchain](https://en.wikipedia.org/wiki/GNU_toolchain) based on the [Mingw-w64](http://mingw-w64.org) and the [mingw-builds](https://github.com/niXman/mingw-builds) project.
+The goal of *mingwpy* project is to provide a free toolchain for building Python extensions for Windows OS, and make sure that those extensions are compatible with CPython compiled with native Visual Studio.
 
-The goal of *mingwpy* project is to provide a free toolchain for building Python extensions for  Windows OS, and make those extensions compatible with CPython compiled with native Visual Studio.
+*mingwpy* toolchain contains customized GCC compiler of the [GNU toolchain](https://en.wikipedia.org/wiki/GNU_toolchain) based on the [Mingw-w64](http://mingw-w64.org) and the [mingw-builds](https://github.com/niXman/mingw-builds) projects.
```
Comment (ogrisel) [inline README.md:3]: gcc, g++ and gfortran compilers

Comment (techtonik) [inline README.md:3]: Fixed in #16.

**Pull request merged**